### PR TITLE
fix: NO-JIRA remove circle button class when its not an icon

### PR DESCRIPTION
**Pull request by sanjeevdialpad**
# Message input having circular send button by default

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Description

 Message input having circular send button by default. But when its not an icon, we need it not to be circle by default. the consumer can add this if required or it needs not be the default goto option.

## :bulb: Context

avoiding this:

Screenshot 2024-04-09 at 15 09 19

## :pencil: Checklist

For all PRs:

- [ ] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [ ] I have reviewed my changes.
- [ ] I have added all relevant documentation.
- [ ] I have considered the performance impact of my change.

For all Vue changes:

- [ ] I have added / updated unit tests.
- [ ] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script.
- [ ] I have validated components with a screen reader.
- [ ] I have validated components keyboard navigation.

For all CSS changes:

- [ ] I have used design tokens whenever possible.
- [ ] I have considered how this change will behave on different screen sizes.
- [ ] I have visually validated my change in light and dark mode.
- [ ] I have used gap or flexbox properties for layout instead of margin whenever possible.

If new component:

- I am exporting any new components or constants:
  - [ ] from the index.js in the component directory.
  - [ ] from the index.js in the root (either `packages/dialtone-vue2` or `packages/dialtone-vue3`).
- [ ] I have added the styles for the new component to the `packages/dialtone-css` package.
- [ ] I have created a page for the new component on the documentation site in `apps/dialtone-documentation`.
- [ ] I have added the new component to `common/components_list.cjs`
- [ ] I have created a component story in storybook
- [ ] I have created story / stories for any relevant component variants in storybook
- [ ] I have created a docs page for the component in storybook.
- [ ] I have checked that changing all props/slots via the UI in storybook works as expected.

### CODE REVIEW

File: packages/dialtone-vue3/recipes/conversation_view/message_input/message_input.vue [hunk 1]
```diff
@@ -178,9 +178,9 @@
               kind="default"
               importance="primary"
               :class="[
-                'd-btn--circle',
                 {
                   'message-input-button__disabled d-fc-muted': isSendDisabled,
+                  'd-btn--circle': !showSend.icon,
```
Comment (braddialpad) [inline packages/dialtone-vue3/recipes/conversation_view/message_input/message_input.vue]: Make sure to make the change in vue 2 as well.

**Comment by iropolo** [comment]
Waiting for vue 2 update to approve 👍🏼

**Comment by sanjeevdialpad** [comment]
done. sorry delayed

**Comment by iropolo** [comment]
Before:
image


After:
image


Is this expected change?

**Comment by braddialpad** [comment]
> Before: image
> 
> After: image
> 
> Is this expected change?

I think not, it is only supposed to be square when there is text in it.

**Comment by sanjeevdialpad** [comment]
Fixed my bad, i was using text instead of icon before this change and show mixed it up.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**